.rubocop.yml

```
@@ -22,3 +23,7 @@ Layout/DotPosition:
 Layout/LineLength:
   Max: 120
   Enabled: true
+
+Style/FrozenStringLiteralComment:
+  Exclude:
+    - test/sandbox/app/components/jbuilder_component.json.jbuilder # not yet supported inside jbuilder templates
```

What about supporting any future .jsbuilder file as well?

```suggestion
    - **/*.jbuilder # not yet supported inside jbuilder templates
```